pom.xml

```diff
@@ -124,12 +124,12 @@
             <dependency>
                 <groupId>io.netty</groupId>
                 <artifactId>netty-codec</artifactId>
-                <version>4.1.77.Final</version> <!--e2e tests and samples only-->
+                <version>4.1.86.Final</version> <!--e2e tests and samples only-->
             </dependency>
             <dependency>
                 <groupId>io.netty</groupId>
                 <artifactId>netty-codec-http</artifactId>
-                <version>4.1.77.Final</version> <!--e2e tests and samples only-->
```

Comment:
Do we need to update "netty-codec-http2" and "reactor-netty-http" versions as well?

Reply:
It seems like the gate is passing without upgrading the versions of those packages.

Reply:
I'd update "netty-codec-http2", just so it follows the "netty-codec-http" version. (netty-codec-http2 has a dependency on netty-codec-http, so it should get the vulnerability patch automatically).

@timtay-microsoft - is the http2 lib required for using HTTP2.0?

Reply:
Ah sure! Thanks for checking!

Reply:
Just upgrading the vulnerable package is fine here

Reply:
Upgrading netty packages has caused some issues with our Azure Core dependencies in the past, so upgrading everything may cause some unusual test failures

Reply:
I upgraded the two packages and the gate passed. Am I good to merge it in?

Reply:
Yeah, go ahead